index.js

```diff
@@ -22,95 +22,112 @@ const playerThreeScore = document.querySelector('.player-three-score');
 let playerOne;
 let playerTwo;
 let playerThree;
+let newGame;
+let question;
+let dailyDouble;
 
 startBtn.addEventListener('click', playAudioIntro);
 
+function submitConditional(answer) {
+  if (playerOne.active) {
+    playerOne.submitAnswer(answer)
+  } else if (playerTwo.active) {
+    playerTwo.submitAnswer(answer)
+  } else if (playerThree.active) {
+    playerThree.submitAnswer(answer)
+  }
+}
+
 submitBtn.addEventListener('click', function(e) {
   e.preventDefault();
   if (hostPrompt.innerText.includes('Player One')) {
-    playerOne = new Player(undefined, undefined, undefined, 1, userInput.value);
+    playerOne = new Player(undefined, undefined, undefined, 1, userInput.value, true);
+    newGame.players.push(playerOne);
     userInput.value = '';
     domUpdates.updateHostPrompt('Player Two, enter your name!');
   } else if (hostPrompt.innerText.includes('Player Two')) {
     playerTwo = new Player(undefined, undefined, undefined, 2, userInput.value);
+    newGame.players.push(playerTwo);
     userInput.value = '';
     domUpdates.updateHostPrompt('Player Three, enter your name!');
   } else if (hostPrompt.innerText.includes('Player Three')) {
     playerThree = new Player(undefined, undefined, undefined, 3, userInput.value);
     userInput.value = '';
+    newGame.players.push(playerThree);
     domUpdates.updateNamesAndScores()
     domUpdates.updateHostPrompt(`${playerOne.playerName}, pick a category dollar amount`);
```

Comment:
Lots and lots of repeat code in these if/else if blocks as well. I'd definitely go back and refactor this when you have a chance.